core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -291,6 +291,18 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
     if (proxyUser != null && principal != null) {
       SparkSubmit.printErrorAndExit("Only one of --proxy-user or --principal can be provided.")
     }
+
+    val executorTimeoutThreshold = Utils.timeStringAsSeconds(
+      sparkProperties.getOrElse("spark.network.timeout", "120s"))
+    val executorHeartbeatInterval = Utils.timeStringAsSeconds(
+      sparkProperties.getOrElse("spark.executor.heartbeatInterval", "10s"))
```

Comment:
Rather than repeat a default, just perform the check if both are set.

Reply:
Since some one may only set spark.executor.heartbeatInterval without setting spark.network.timeout.So i just get with default value.Does this make sense?Since we may not need to check if both are not set. Thanks!

Reply:
OK true. This still seems like the wrong place to validate, as they're not arguments to spark-submit, and this requires duplicating default values. @vanzin where would you put this check?

Reply:
I have thought about add a config constant,but it will affect many other codes,so i simply change here. @srowen @vanzin

Reply:
This is the wrong place for the check, see discussion.

Reply:
Done @vanzin Thanks